### DISCUSSION TITLE
[Feature Request] [dialog] make the overlayController support multiple invokers

### DISCUSSION THREAD

**Discussion by T-Motion** [ing-bank/lion#1095] [Ideas] [answered]
Opening a dialog component from multiple invokers is difficult. Changing the "opened" property of a dialog will not trigger the accessibility features.

Proposal, add following functionalities to the OverlayMixin:
- open()
- close()
- toggle()

**Comment by tlouisse** [comment, marked as answer]
The pubblic interface is added here: https://github.com/ing-bank/lion/pull/1103
